src/test/README.md

```diff
@@ -52,4 +52,4 @@ examine `uint256_tests.cpp`.
 
 For further reading, I found the following website to be helpful in
 explaining how the boost unit test framework works:
-[http://www.alittlemadness.com/2009/03/31/c-unit-testing-with-boosttest/](http://www.alittlemadness.com/2009/03/31/c-unit-testing-with-boosttest/).
```

Comment:
Nice catch!

How about pointing to official documentation? Also, this paragraph could be improved to remove the first person.

Reply:
I don't think he has to fix all the surrounding issues in the file just for fixing a link.

IMO this PR is fine as-is. But if you want to make further updates sure!